static/chicago-immigration-law.html

```diff
@@ -0,0 +1,912 @@
+            <h4>Best lawyer</h4>
+            <p>Keith has help me so much with the process by far the best you can trust him wither he process he is well knowledgeable and knows what he is doing highly recommends him</p>
+            <div class="user">
+              <ion-icon name="ios-person" size="large" role="img" class="md icon-large hydrated" aria-label="person"></ion-icon>
+              <p><span>Musa</span> client</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="item">
+        <img src="/img/keyboard-bg2.jpg" alt="keyboard">
+        <div class="carousel-caption">
+          <div>
+            <img src="/img/cotation.png" alt="IMG">
+            <h4>Highly recommend</h4>
+            <p>Keith helped my company apply for a visa for a remote worker. The situation was nuanced and tricky and he went above and beyond to ensure the outcome was a success. He was very knowledgeable, responsive, and reassuring throughout the application process. Highly recommended.</p>
+            <div class="user">
+              <ion-icon name="ios-person" size="large" role="img" class="md icon-large hydrated" aria-label="person"></ion-icon>
+              <p><span>Ali</span> client</p>
+            </div>
+          </div>
+        </div>
+      </div>
+  
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+</div>
+
+      </div>  <!-- end col-md-8 -->
+      <div class="col-md-2">
+          &nbsp;
+      </div>
+    </div>  <!-- end row -->
+  </div>  <!-- end container -->
+  </section>
+
+  <section id="lp-form" style="background: #9d6667;">
+    <div class="container">
+      <div class="row">
+        <div class="text-center">
+          <h2 style="color: #fff;">Let's Talk!</h2>
+          <p class="text-center txt-feature txt-white">Let Us Know How Can We Help</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-2">
+          &nbsp;
+        </div>
+        <div class="col-md-8">
+          <form name="contact" method="POST" data-netlify="true">
+            <label for="fname">Name *</label>
+            <input type="text" id="fname" name="name" required="" >
+            <label for="femail">Email *</label>
+            <input type="email" id="femail" name="email" required="" >
+            <label for="fphone">Phone</label>
+            <input type="text" id="fphone" name="phone" >
+            <label for="fmsg">Message *</label>
+            <textarea id="fmsg" name="message"></textarea>
```

Comment:
```suggestion
            <textarea id="fmsg" name="message" required=""></textarea>
```

This one is required as well? Or should we take off the `*` from the label?